spec/models/miq_event_definition_spec.rb

```diff
@@ -27,18 +27,19 @@
       let!(:set1) { create_set!('host_operations') }
       let!(:set2) { create_set!('evm_operations') }
 
-      context 'and CSV file with an event definition linked to one of the sets' do
-        let(:csv) do
-          <<-CSV.strip_heredoc
-            name,description,event_type,set_type
-            #
-            evm_server_start,EVM Server Start,Default,host_operations
-          CSV
+      context 'and YML file with an event definition linked to one of the sets' do
+        let(:yml) do
+          <<-YML.strip_heredoc
+            - name: evm_server_start
+              description: EVM Server Start
+              event_type: Default
+              set_type: host_operations
+          YML
```

Comment:
Not sure if this is right but I tried to update the test cases but they are still failing with the same error.